src/library/scala/concurrent/ExecutionContext.scala

```diff
@@ -166,6 +166,25 @@ object ExecutionContext {
     override final def reportFailure(t: Throwable): Unit = defaultReporter(t)
   }
 
+  /**
+   * This `ExecutionContext` is ideally suited to execute short-lived tasks on the `global` `ExecutionContext` as it
+   * attempts to locally batch the execution of nested tasks.
```

Comment:
we should add a hint how this can be used (once we've settled on it, see my other comment)

Reply:
Good point—let's settle that first, then revisit this.